app/frontend/src/service/client.ts

```diff
@@ -8,9 +9,36 @@ import { SSOPromiseClient } from "../pb/sso_grpc_web_pb";
 
 const URL = process.env.REACT_APP_API_BASE_URL;
 
+let _unauthenticatedErrorHandler: (e: object) => void = () => {};
```

Comment:
```suggestion
let _unauthenticatedErrorHandler: () => Promise<void> = () => {};
```
Wouldn't this be fine, since nothing gets passed through the useEffect callback? Also the return type should match the one in the setter?

Reply:
I was passing the error anyway in case for future use, but yeah it won't be necessarily.